Gopkg.toml

```diff
@@ -30,7 +30,7 @@
 
 [[constraint]]
   name = "github.com/weaveworks/common"
-  source = "https://github.com/tomwilkie/weaveworks-common"
+  source = "https://github.com/sandlis/weaveworks-common"
```

Comment:
Instead of keeping it under personal accounts umbrella, is it feasible to move the fork under the Grafana organization?

Reply:
The `weaveworks-common` PR should get merged soon, after which we can update the dependencies again. Many of us are depending on this vendor update for a bug fix or use newer Cortex features, so we couldn't wait for the `weaveworks-common` PR to get merged.

Reply:
https://github.com/grafana/loki/issues/945 to keep track of it.